helm/hpcc/templates/sasha.yaml

```diff
@@ -3,7 +3,11 @@
 {{ if not $sasha.disabled -}}
 {{ $_ := set $sasha "name" $sashaName -}}
 {{ $_ := set $sasha "access" (splitList " " (include "hpcc.getSashaServiceAccess" $sasha)) -}}
-{{ $commonCtx := dict "root" $ "me" $sasha -}}
+{{ $secretsCategories := list "system" }}
+{{ if has "data" $sasha.access }}
+{{ $secretsCategories := append $secretsCategories "storage" }}
```

Comment:
I think there are newline issues here, each of the above lines will introduce a newline.
Some newline suppression needed with {{- or -}} , being careful not to swallow up the preceding or final newline.

Try looking at the output of helm template (or can use startall.sh -t) in this area, to see newlines.

Reply:
This section didn't generate extra newlines.  However, I have added {{- and -}} to these lines.  I have found the helper functions creating extra new lines.  I have fixed the ones related to secrets.  However, there are many other helper functions that do generate newlines unnecessarily.  I've created a ticket to address these: https://track.hpccsystems.com/browse/HPCC-25037

Reply:
ok, thanks.
I think these lines didn't generate a newline because of the pre-existing line -}} on line 5 chomped the newlines they'd otherwise introduce.